collect_app/src/main/java/org/odk/collect/android/activities/CrashHandlerActivity.kt

```diff
@@ -0,0 +1,28 @@
+package org.odk.collect.android.activities
+
+import android.os.Bundle
+import androidx.activity.OnBackPressedCallback
+import org.odk.collect.crash_handler.CrashHandler
+import org.odk.collect.strings.localization.LocalizedActivity
+
+class CrashHandlerActivity : LocalizedActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val crashHandler = CrashHandler.getInstance(this)!!
```

Comment:
Not usually a fan of `!!`, but we really shouldn't be getting to this Activity if the `CrashHandler` instance is `null`.

Reply:
I'm still thinking about it... can't we make sure that `CrashHandler` always exist? We always should have that object available right?

Reply:
Currently, no. It wouldn't be available before calling `install` and it really shouldn't be available after calling `uninstall` (I'll make that change now). As I said in https://github.com/getodk/collect/pull/5324#discussion_r1017838264, we currently use the fact that it can be nulled out by clearing app state as well.